packages/slider/src/SliderHandle.ts

```diff
@@ -170,4 +177,31 @@ export class SliderHandle extends Focusable {
         if (!this.formatOptions) return;
         return this.getNumberFormat();
     }
+
+    public connectedCallback(): void {
+        super.connectedCallback();
+        this.resolveLanguage();
+    }
+
+    public disconnectedCallback(): void {
+        this.resolveLanguage();
+        super.disconnectedCallback();
+    }
+
+    private resolveLanguage(): void {
+        const queryThemeEvent = new CustomEvent<ProvideLang>(
+            'sp-language-context',
+            {
+                bubbles: true,
+                composed: true,
+                detail: {
+                    callback: (lang: string) => {
+                        this.resolvedLanguage = lang;
+                    },
+                },
+                cancelable: true,
+            }
+        );
+        this.dispatchEvent(queryThemeEvent);
+    }
```

Comment:
I wonder if these, and the property declarations, should eventually become a mixin?

Reply:
Something, yes. I've been putting of code share refactors for post `lit@2.0` so we can properly evaluate the Controller pattern as an option along side mixins.